### ISSUE TITLE
Are there any plans on original .msstyle files support?

### ISSUE THREAD

**Issue by banq007** [rozniak/xfce-winxp-tc#222]
This project already looks great! But with native win xp theme support it would be truly mindblowing.

**Comment by rozniak** [comment]
Yeah I have had it in mind to eventually support generating GTK/XFWM theme - most likely something I'll look into further during work on #122 . With the parts based system I did for GTK it hopefully isn't too bad to sort out most things.

Blocked by #216 which would make generating XFWM themes significantly better.

**Comment by dominichayesferen** [comment]
I'm gonna assume you did, but when scaling non-DWM controls make sure they are nearest pixel/neighbour scaled - it was always that method prior to DWM happening and inheriting DirectX's bilinear scaling methods in titlebars.
